pkgs/development/r-modules/build_packages.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
```

Comment:
What is the purpose of this script?

Reply:
It's a test script to build packages. I used it to test all of the packages. Complements the generation of all the packages created by generate-r-packages.R. Intended for use by someone updating all the R packages, not a user trying to install some.